app/src/main/java/org/wikipedia/games/onthisday/OnThisDayGameViewModel.kt

```diff
@@ -198,13 +198,11 @@ class OnThisDayGameViewModel(savedStateHandle: SavedStateHandle) : ViewModel() {
             "game_end"
         } else if (_gameState.value is Resource.Loading) {
             "game_loading"
-        } else {
-            if (::currentState.isInitialized.not()) {
-                "game_loading"
-            } else
-                "game_play_" + (currentState.currentQuestionIndex + 1)
+        } else if (::currentState.isInitialized.not()) {
+            "game_loading"
+        } else
+            "game_play_" + (currentState.currentQuestionIndex + 1)
```

Comment:
Is it possible to combine `::currentState.isInitialized.not() || _gameState.value is Resource.Loading`?

```
fun getCurrentScreenName(): String {
      return if (_gameState.value is GameEnded) {
          "game_end"
      } else if (::currentState.isInitialized.not() || _gameState.value is Resource.Loading) {
          "game_loading"
      } else {
          "game_play_" + (currentState.currentQuestionIndex + 1)
      }
  }
```